src/constants/common.js

```diff
@@ -21,4 +21,9 @@ module.exports = {
 	},
 	internalAccessUrls: [],
 	SCP_SERVICE: 'scp',
+	RESOURCE_LIST: ['project', 'observation_with_rubrics', 'observation', 'survey'],
+	REVIEW_TYPE: {
```

Comment:
@adithyadinesh0412 define string instead of object so you can check for the conditions